tests/fixture/e2e/env.go

```diff
@@ -158,6 +159,10 @@ func NewTestEnvironment(tc tests.TestContext, flagVars *FlagVars, desiredNetwork
 		}, DefaultTimeout, DefaultPollingInterval, "failed to see all chains bootstrap before timeout")
 	}
 
+	if flagVars.StartNetwork() {
+		os.Exit(0)
+	}
```

Comment:
Ah so this prevents any tests from running even if starting with ginkgo by using `os.Exit(0)` directly

This should be much better than our previous way of using `ginkgo.focus` to run a single test (simple ping)